L77-core-cpp-third-party-identity-support-for-call-credentials.md

```diff
@@ -76,32 +76,47 @@ This doc proposes to extend the access to Google APIS directly with 3p identitie
 
 **Implicit flow**
 
-In core, the underlying credentials retrieval process of `grpc::GoogleDefaultCredentials()` will be extended to accept 3p identity credentials implicitly.
+In core, `grpc::GoogleDefaultCredentials()` follows the ADC (Application Default Credentials) credentials retrieval process. It will be extended to accept 3p identity credentials. More information on ADC and how they work can be found here: https://developers.google.com/identity/protocols/application-default-credentials
 
 **Explicit flow**
 
-In core and cpp, a public api will be added to create an `ExternalAccountCredentials` with explicit 3p identity credentials.
-```
+The downside with `GoogleDefaultCredentials()` is that customers will have to use a json credentials file. So besides the implicit flow, in core and C++, a public api will be added to create an `ExternalAccountCredentials` with explicit 3p identity credentials.
```

Comment:
The wording here incorrectly implies that the core API is a public API, which is incorrect.

I suggest replacing "in core and C++, a public api will be added" with "a public API will be added in C++".

Reply:
The comment was addressed in https://github.com/grpc/proposal/pull/221